front/vue.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
       .set('@styles', path.resolve('src/styles/'))
       .set('@layout', path.resolve('src/layouts/'))
       .set('@locales', path.resolve('src/locales/'))
+      .set('@assets', path.resolve('src/assets/'))
```

Comment:
don't we have src/assets automatically? do we need to add it manually? i have big doubts we have to

Reply:
I had to add it manually, because without it, the path was not found. but it's possible that I did something wrong